### ISSUE TITLE
colorama.ansi.set_title('') should (un|re)set title

### ISSUE THREAD

**Issue by Zearin** [tartley/colorama#62]
I just noticed that in `colorama.ansi`, `set_title` doesn’t do anything if you give it an empty string.  Therefore, if you use it in a Python program, when that program exits, the title does not return to whatever it was before.  However, if I type in `echo -n -e "\033]0;\007"`, it resets to its default value.  (I’m on OS X 10.10, using Terminal.app, so the default value is simply “Terminal”.)

So, I think colorama should:

- allow using `set_title` with the empty string
- offer a keyword option (in `colorama.init()` or in `ansi.set_title()`) to unset the window title when Python exits

**Comment by tartley** [comment]
What if the user wanted to set the title to an empty string?

Might None be a more appropriate value to return the title to default?

I think the application should have to remember the initial title if it wants to reset the title to what it was before.

**Comment by wiggin15** [comment]
colorama is trying to emulate the bahvior of unix terminals - so if unix terminals reset the title to the original value when passed an empty string maybe colorama should do that too.
I'm not sure it's worth the trouble though.

**Comment by tartley** [comment]
> colorama is trying to emulate the bahvior of unix terminals

Yeah, that philosophy does make sense. I retract my 'None' suggestion.
